app/feeswapper/management/commands/feeSwapper.py

```diff
@@ -0,0 +1,185 @@
+import json
+import time
+
+from django.conf import settings
+from django.core.management.base import BaseCommand, CommandError
+from django.utils.timezone import now
+
+import requests
+import sendgrid
+from feeswapper.models import CurrencyConversion
+from sendgrid.helpers.mail import Content, Email, Mail
+from web3 import Web3
+from web3.middleware import geth_poa_middleware
+
+# Amount of slippage from target Ether price from estimated price on exchange allowed when trading tokens back to ETH
+SLIPPAGE = 0.05  
```

Comment:
Put this in settings.py

Reply:
Done locally.  Will include in next commit.